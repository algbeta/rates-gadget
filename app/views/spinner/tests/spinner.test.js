import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../index';

describe('Spinner tests', () => {
  test('Spinner renders correctly', () => {
    const tree = renderer.create(<Spinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
