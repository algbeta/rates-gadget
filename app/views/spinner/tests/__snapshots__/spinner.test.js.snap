// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner tests Spinner renders correctly 1`] = `
<div
  className="spinner-container"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
